src/pages/kurzy/[slug]/index.tsx

```diff
@@ -257,6 +269,16 @@ const CourseDetailContent = ({
               )}
             </Flex>
           </Flex>
+          <StyledSidebarCourseReviews courseOverview={courseOverview} />
+          {courseOverview.reviewsOverview.numberOfReviews > 0 && (
+            <AllReviewssText
```

Comment:
Another huge mistake... ❗ 
```suggestion
            <AllReviewsText
```